integrationTests/DonationPersistenceTest.php

```diff
@@ -104,6 +104,7 @@ private function donationRow(): array
             'preAuthorizationDate' => null,
             'mandate_id' => null,
             'mandateSequenceNumber' => null,
+            'tipRefundAmount' => null,
```

Comment:
Would be good to sync `api.yaml` to make it easier to find the expected type when working across apps.

Reply:
Although we could add the new property into the API that FE uses - either now or at the time we need it. We could start to usefully display it since we now let people view old donations on the site.

Reply:
@bdsl My preference is to use it as the description currently suggests:
> As Salesforce & MatchBot share a Donation schema, endpoints for both are documented here
    together; however operation `salesforceUpsert` is implemented only in Salesforce
    and all others are implemented only in MatchBot.